README.md

```diff
@@ -12,9 +12,11 @@ OpenZeppelin Stellar Soroban Contracts is a collection of contracts for the Stel
 
 ## Project Structure
 
-- `contracts/`: Source code
+- `packages/`: Source code
   - `tokens/`: Various token types (fungible, non-fungible, etc.)
-  - `utils/`: Utilities for token types (pausable, etc.)
+  - `contract-utils/`: Utilities for token types (pausable, etc.)
+  - `constants/`: Time-based constants
```

Comment:
Instead of:
> Time-based constants

I'd go for:
> Constants shared across token standards

Because we may add other constants that are not time related in the future